src/marks/auto.js

```diff
@@ -90,7 +90,16 @@ export function autoSpec(data, options) {
       colorMode = "stroke";
       break;
     case "line":
-      markImpl = X && Y ? line : X ? lineX : lineY; // 1d line by index
+      markImpl =
+        X && Y // same logic as area (see below), but default to line
+          ? yZero || isMonotonic(X)
+            ? lineY
+            : xZero || isMonotonic(Y)
+            ? lineX
+            : line
+          : X // 1d line by index
+          ? lineX
+          : lineY;
```

Comment:
```suggestion
          ? lineY
          : lineX;
```

i'd also want to add this test

```
export async function autoLineHistogramY() {
  const aapl = await d3.csv<any>("data/aapl.csv", d3.autoType);
  return Plot.auto(aapl, {y: "Volume", mark: "line"}).plot({marginLeft: 80});
}
```

Reply:
I see the problem, but I don’t think that’s the right fix. I think the additional context in this example is that there’s a reducer for _y_, and so even those only _x_ is provided as input, _x_ is binned, effectively meaning that _x_ is monotonic (and _y_ is derived). So, we need to consider the reducers when determining the mark implementation, too.

Reply:
Okay, I think the latest commit fixes the problem.